src/main/java/com/microsoft/azure/relay/ClientWebSocket.java

```diff
@@ -152,7 +152,9 @@ boolean isOpen() {
 	 */
 	CompletableFuture<String> readTextAsync() {
 		return this.textQueue.dequeueAsync().thenApply(text -> {
-			RelayLogger.logEvent("receivedText", this, String.valueOf(text.length()));
+			if (text != null) {
```

Comment:
>text [](start = 7, length = 4)

Does a null text ever happen? Maybe at close time? Do you want any trace then?

Reply:
That's the reason why I put a null check. Text can be null after close and text.length() will give a null pointer.

---
In reply to: [261783196](https://github.com/Azure/azure-relay-java/pull/27#discussion_r261783196) [](ancestors = 261783196)